documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -909,6 +916,16 @@ Lucene In Action from Otis Gospodnetic and Erik Hatcher.
 [[section-dynamic-boost]]
 ==== Dynamic index time boosting
 
+[WARNING]
+====
+Index-time boosting is deprecated: it will be removed starting from Lucene 7.
+
+To implement static boosting, you can use query-time boosting instead: see <<search-query>>.
```

Comment:
As a matter of fact, query-time boosting through our DSL is "static" (at least if we give "static" the same meaning as in the "Static index time boosting" section above): a given field will have the same boost for every document, regardless of the content of each document.

Anyway... I rephrased this so that I don't mention "static" at all.